lib/bundler/templates/newgem/bin/setup.tt

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
+cd -
```

Comment:
I'm sorry but I still don't understand why changing directories at all is necessary here